dd-bridge-android/src/main/kotlin/com/datadog/android/bridge/internal/BridgeSdk.kt

```diff
@@ -20,18 +22,26 @@ internal class BridgeSdk(context: Context) : DdSdk {
     internal val appContext: Context = context.applicationContext
 
     override fun initialize(configuration: DdSdkConfiguration) {
-        val configBuilder = DatadogConfig.Builder(
+        val credentials = Credentials(
```

Comment:
I was just thinking...how we will be able to configure the `source` for all our events. Right now the `source` it is `android` and it is a constant value. Probably we will have to make it configurable also to be able to differentiate between React/Android events.